src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -14,9 +14,11 @@ describe('telemetryBuildFinishedAction', () => {
 
   beforeEach(() => {
     sys = createSystem();
-    config = mockValidatedConfig(sys);
-    config.outputTargets = [];
-    config.flags.args = [];
+    config = mockValidatedConfig({
+      flags: { args: [], knownArgs: [], task: 'build', unknownArgs: [] },
```

Comment:
this could be a `createConfigFlags` call here if we wanted, just to clean things up a little bit

Reply:
👍 Done in 57d6a3bb8